app/models.py

```
@@ -36,6 +37,8 @@ class Config:
                 "entry_message": "Hello! I'm here to assist with the meeting.",
                 "enable_tools": True,
                 "extra": {"company": "ACME Corp", "meeting_purpose": "Weekly sync"},
+                "llm_prompt": "You are Meeting Assistant, a concise and professional \
+                AI bot that helps summarize key points and keep the meeting on track. Speak clearly and stay on topic."
```

Pylint (3.3.7)

[convention] Line too long (119/100)

(C0301)